src/Servers/Kestrel/Core/src/KestrelServerLimits.cs

```diff
@@ -42,6 +42,9 @@ public class KestrelServerLimits
     private long? _maxConcurrentConnections;
     private long? _maxConcurrentUpgradedConnections;
 
+    // Matches the default IIS minBytesPerSecond
+    private MinDataRate? _minRequestBodyDataRate = new MinDataRate(bytesPerSecond: 240, gracePeriod: TimeSpan.FromSeconds(5));
```

Comment:
???

Reply:
This isn't for realz. I'm just adding a change that should break the gRPC interop tests, demonstrating they run against the just-built Microsoft.AspNetCore.App shared Fx bits.